pytealext/evaluator/evaluator.py

```diff
@@ -486,13 +499,25 @@ def eval_teal(
             b = stack.pop()
             a = stack.pop()
             if a != 0:
-                raise Panic(
+                raise Exception(
```

Comment:
Exception describes our evaluator's limitation, Panic resembles a runtime panic in TEAL